config/operator/crd/bases/scylla.scylladb.com_scyllaclusters.yaml

```diff
@@ -1618,6 +1618,35 @@ spec:
                 - name
                 type: object
               type: array
+            upgrade:
+              description: UpgradeStatus contains state of ongoing upgrade procedure.
+              properties:
+                currentNode:
+                  description: CurrentNode node under upgrade.
+                  type: string
+                currentRack:
+                  description: CurrentRack rack under upgrade.
+                  type: string
+                dataSnapshotTag:
+                  description: SystemSnapshotTag snapshot tag of data keyspaces.
+                  type: string
+                fromVersion:
+                  description: FromVersion reflects from which version ScyllaCluster is being upgraded.
+                  type: string
+                state:
+                  description: State reflect current state machine state.
+                  type: string
+                systemSnapshotTag:
```

Comment:
Why do we need the snapshots?

Reply:
In order to restore tables in case of failure. Rollback requires restoring system tables to state prior upgrade.